pkg/k8shandler/kibana_handler/configmap.go

```diff
@@ -0,0 +1,106 @@
+package kibana_handler
```

Comment:
configmaps should be common for both ES and Kibana... i would add necessary functions to the `k8shandler/configmaps.go` file

Reply:
I thought about that. But the current state of the packaging would create a package that depends on another package (kibana will be importing elasticsearch). Can we leave that for now until the restructuring of the `k8shandler` package?

Reply:
In the interest of other priorities, I would prefer yet adding this to the TD pile

Reply:
I think this ended up being a file that was copied and pasted from CLO (as with other files in the package `kibana_handler`)